diff -ruN a/Makefile.am b/Makefile.am
--- a/Makefile.am 2022-09-07 16:05:25.148636282 +0800
+++ b/Makefile.am 2022-09-07 16:05:25.156636177 +0800
@@ -195,7 +195,7 @@
 bin_SCRIPTS = curl-config

 SUBDIRS = lib src
-DIST_SUBDIRS = $(SUBDIRS) tests packages scripts include docs
+DIST_SUBDIRS = $(SUBDIRS) packages include

 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libcurl.pc
@@ -306,8 +306,6 @@
 # We extend the standard install with a custom hook:
 install-data-hook:
 	(cd include && $(MAKE) install)
-	(cd docs && $(MAKE) install)
-	(cd docs/libcurl && $(MAKE) install)

 # We extend the standard uninstall with a custom hook:
 uninstall-hook:
diff -ruN a/Makefile.in b/Makefile.in
--- a/Makefile.in 2022-09-07 16:05:24.624643165 +0800
+++ b/Makefile.in 2022-09-07 16:05:24.276647738 +0800
@@ -708,7 +708,7 @@

 bin_SCRIPTS = curl-config
 SUBDIRS = lib src
-DIST_SUBDIRS = $(SUBDIRS) tests packages scripts include docs
+DIST_SUBDIRS = $(SUBDIRS) packages include
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libcurl.pc
 LIB_VAUTH_CFILES = \
@@ -1788,8 +1788,6 @@
 # We extend the standard install with a custom hook:
 install-data-hook:
 	(cd include && $(MAKE) install)
-	(cd docs && $(MAKE) install)
-	(cd docs/libcurl && $(MAKE) install)

 # We extend the standard uninstall with a custom hook:
 uninstall-hook:
